docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -282,4 +284,19 @@ Name of the vault to access. Sub-domain of `vault.azure.net`.
 
 ```bash tab="Example"
 --keyvault-name="MyKeyVault" 
-```
\ No newline at end of file
+```
+
+## Multifile Options
```

Comment:
Does it make more sense to put this directly after the File Options section so the two methods for file-based signing are grouped together?

Reply:
Done.